Makefile

```diff
@@ -9,7 +9,11 @@ build-image:
 	docker build -t sigdocs -f generator/Dockerfile generator
```

Comment:
I wonder if we can give this a more descriptive image name? Like perhaps kube-communitydocs ?